server/global/molecules_importation/moleculesAnalyzer.js

```diff
@@ -173,18 +181,39 @@ function getNonValidNumberValue(molecules) {
   );
 }
 
-export class AnalyzerWarning {
+/**
+ * Normalize a molecule DCI
+ * @param {string} dci
+ */
+export const normalizeDCI = (dci) =>
+  String(dci)
+    .trim()
+    .normalize("NFD")
+    .replace(/[\u0300-\u036f]/g, "")
```

Comment:
Can a comment explain what this regexp is doing?

Reply:
done 👍